api/src/backend/api/v1/serializers.py

```diff
@@ -1113,6 +1113,14 @@ class Meta:
         fields = ["id", "name"]
 
 
+class ScanThreatscoreReportSerializer(serializers.Serializer):
+    id = serializers.CharField(source="scan")
+
+    class Meta:
+        resource_name = "scan-reports"
+        fields = ["id"]
+
+
```

Comment:
Why do we need this? We are returning the following in the endpoint that uses it:

```python
def _serve_file(self, content, filename, content_type):
    response = HttpResponse(content, content_type=content_type)
    response["Content-Disposition"] = f'attachment; filename="{filename}"'

    return response

return self._serve_file(content, filename, "application/pdf")
```